lib/SetAPI/rnaseq_set_functions.py

```diff
@@ -0,0 +1,128 @@
+"""Functions for retrieving various RNASeq sets from the DB."""
```

Comment:
Moved the fake set wrangling to a separate file -- the methods were c&p'd from the individual classes.